openapi.yaml

```diff
@@ -94,12 +84,24 @@ parameters:
     required: true
     description: OSU ID of a person
     pattern: '^\d{9}$'
-  applicationId:
-    name: applicationId
-    in: path
-    type: string
-    required: true
-    description: A unique ID of a application
+  processDocumentBody:
+    name: processDocumentBody
+    in: body
+    schema:
+      type: object
+      properties:
+        onbaseDocumentType:
+          type: string
+          description: OnBase document type
+        additionalChecklistInfo:
+          type: string
+          description: Additional checklist information
+        documentReceiveDate:
+          type: string
+          format: date-time
+          description: The document receive date in Oracle date format (DD-MON-YYYY HH24:MI:SS)
+          example: '20-MAY-19 00:00:00'
```

Comment:
For this and all other properties that follow this Oracle date format, we need to remove `format: date-time`. Otherwise, express-openapi will try to validate these properties against the rfc 3339 (iso 8601) date-time format. If we want to do validation on these parameters, we will need to use `pattern`

Reply:
Yes, I missed this as well, will fix.